recipe/meta.yaml

```diff
@@ -36,6 +42,8 @@ outputs:
       run_constrained:
         - cuda-version >=11.2,<12  # [(cuda_compiler_version or "None").startswith("11")]
         - cuda-version >=12,<13    # [(cuda_compiler_version or "None").startswith("12")]
+        - cudatoolkit              # [(cuda_compiler_version or "None").startswith("11")]
+        - cuda-cudart              # [(cuda_compiler_version or "None").startswith("12")]
```

Comment:
If there is no constraint on the versions here, it doesn't make a difference at all.

Reply:
No, there is a difference. We want users to explicitly use `cuda-version` to control the version of the installed `cudatoolkit` (CUDA<=11) or `cuda-cudart` (CUDA >=12). We let it float here to pass the control to user as per [`cuda-version`'s guideline](https://github.com/conda-forge/cuda-version-feedstock/blob/main/recipe/README.md).

Reply:
Yeah think you are both right. There are differences between the cases, but in the end `run_constrained` probably doesn't help the user as much as we might like

Think it is still be worth keeping for documentation/clarity